ipatests/prci_definitions/nightly_ipa-4-10_latest.yaml

```diff
@@ -1649,9 +1649,21 @@ jobs:
       class: RunADTests
       args:
         build_url: '{fedora-latest-ipa-4-10/build_url}'
-        test_suite: test_integration/test_trust.py
+        test_suite: test_integration/test_trust.py::TestTrust
         template: *ci-ipa-4-10-latest
-        timeout: 10000
+        timeout: 6000
+        topology: *adroot_adchild_adtree_master_1client
+
+  fedora-rawhide/test_trust_autoprivate:
```

Comment:
The job name must start with `fedora-latest-ipa-4-10` instead of `fedora-rawhide`.